src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/TriviaEngine/LineColumnRule.cs

```diff
@@ -120,13 +120,13 @@ public static LineColumnRule ForceSpacesOrUseDefaultIndentation(int spaces)
                 spaces,
                 indentation: -1);
 
-        public static LineColumnRule ForceSpacesOrUseAbsoluteIndentation(int spacesOrIndentation)
+        public static LineColumnRule ForceSpacesOrUseFollowIndentation(int spacesOrIndentation)
```

Comment:
💡 This parameter now only applies to indentation

```suggestion
        public static LineColumnRule ForceSpacesOrUseFollowIndentation(int indentation)
```

Reply:
fixed